<launch>
    <!-- Setup parameters -->
    <param name="QUERY_STRATEGY" type="string" value="targeted" />
    <param name="MAIN_FILEPATH" type="string" value="$(find lfd_experiments)/experiment_data/cc_lfd/feedback_demo/old_data"/>
    <param name="DEMO_VOICE_FILEPATH" type="string" value="$(find feedback_cclfd)/demo_data"/>

    <!-- Fake LfD server (for debugging) -->
    <!-- Uncomment the below node and comment over the real node if not using Sawyer -->
    <!-- <node name="fake_lfd_server" pkg="feedback_cclfd" type="fake_lfd_server.py" output="screen" />         -->

    <!-- Launch CC-LfD server -->
    <!-- Comment the below node if using the fake server above -->
    <node name="feedback_demonstration" pkg="lfd_experiments" type="feedback_demonstration_server.py" output="screen" />

    <!-- Launch classifiers -->
    <node name="synthesizer" pkg="feedback_cclfd" type="synthesizer.py" output="screen" />

    <!-- Launch NLP -->
    <node name="nlp_google_server" pkg="cairo_nlp" type="nlp_google_server.py" output="screen" />

    <!-- Launch planning nodes -->
    <node name="query_nlp" pkg="feedback_cclfd" type="query_nlp.py" output="screen" />
    <node name="demonstrator" pkg="feedback_cclfd" type="demonstrator.py" output="screen" />
    <node name="request_feedback_server" pkg="feedback_cclfd" type="request_feedback_server.py" output="screen" />
    <node name="teleop" pkg="feedback_cclfd" type="teleop.py" output="screen" launch-prefix="xterm -e" />

</launch>